packages/dds/tree/package.json

```diff
@@ -149,6 +149,7 @@
 		"@fluidframework/eslint-config-fluid": "^5.3.0",
 		"@fluidframework/test-runtime-utils": "workspace:~",
 		"@fluidframework/test-utils": "workspace:~",
+		"@fluidframework/tree-previous": "npm:@fluidframework/tree@2.0.0",
```

Comment:
I'm not up-to-date on recent changes in typetests, but we probably want this to align with other packages' "previous" install. Looks like client packages are generally still installing 2.0.0-rc.5.0.0 as previous, hence larger-than-expected diff to package-lock.

Reply:
Looks like the other packages need to be updated, as part of our release process is to update the type tests to target the most recent minor version (2.0). I'll land that before landing this.